op-supervisor/supervisor/cli_config.go

```diff
@@ -0,0 +1,42 @@
+package supervisor
+
+import (
+	"errors"
+
+	"github.com/urfave/cli/v2"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum-optimism/optimism/op-service/oppprof"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+)
+
+type CLIConfig struct {
+	Version string
+
+	LogConfig     oplog.CLIConfig
+	MetricsConfig opmetrics.CLIConfig
+	PprofConfig   oppprof.CLIConfig
+	RPC           oprpc.CLIConfig
+
+	// MockRun runs the service with a mock backend
+	MockRun bool
```

Comment:
Probably should set this based on the cli flag?

Reply:
good catch, fixed